FutureGateway/php/17_stdout

```diff
@@ -0,0 +1 @@
+sayhello.sh: line 6: fg-hackfest.sci-gaia.eu: command not found
```

Comment:
Why is there reference to `sayhello.sh` in a php script ? Surely this should be `sayhello.php` ?